### PR TITLE
Add blog post: Running Presto on Alluxio Structured Data Service on Your Laptop

### PR DESCRIPTION
**Pull request by dbw9580**

### CODE REVIEW

File: website/blog/2021-06-04-running-presto-alluxio-sds.md [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+        `./bin/alluxio-mount.sh SudoMount`
+
+        You may need to input your password because mounting RAMFS requires sudo privileges.
+
+    4.  Format Alluxio filesystem:
+
+        `./bin/alluxio format`
+
+        NOTE: This step is only required when you run Alluxio for the first time. If you run this command for an existing Alluxio cluster, all previously stored data and metadata in the Alluxio filesystem will be erased. However, data in under storage will not be changed.
```
Comment (beinan) [inline website/blog/2021-06-04-running-presto-alluxio-sds.md:308]: "data in under storage will not be changed."  I guess you mean the data in the underlying storage would not be changed,  just wanna clarify, is the underlying storage here hive data files or hive-metastore, or both?

**Comment by apc999** [comment]
I found this blog written by @maobaolong and recommended to post it here 
thoughts?

File: website/blog/2021-06-04-running-presto-alluxio-sds.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
+---
+id: running-presto-alluxio-sds
+title: Running Presto on Alluxio Structured Data Service on Your Laptop
+author: Baolong Mao
+authorURL: https://www.linkedin.com/in/baolong-mao-180624136/
+---
+
+**Authors**: Baolong Mao (Tencent), Bowen Ding (Fudan University)
```
Comment (beinan) [inline website/blog/2021-06-04-running-presto-alluxio-sds.md:8]: The format of Authors is not consistent with other multi author blogs.  See this one: https://github.com/prestodb/prestodb.github.io/blob/source/website/blog/2020-06-16-alluxio-datacaching.md

**Review by beinan — APPROVED** [review]
Looks good to me, just a couple format and clarification question

File: website/blog/2021-06-04-running-presto-alluxio-sds.md (hunk 1 above)
Comment (dbw9580) [reply website/blog/2021-06-04-running-presto-alluxio-sds.md:308]: I think both won't be affected.

File: website/blog/2021-06-04-running-presto-alluxio-sds.md (hunk 2 above)
Comment (dbw9580) [reply website/blog/2021-06-04-running-presto-alluxio-sds.md]: Fixed in 5320f87.

Comment (beinan) [reply website/blog/2021-06-04-running-presto-alluxio-sds.md]: Thank you!

**Comment by apc999** [comment]
@aweisberg can you take a look? Thanks!

**Comment by apc999** [comment]
ping @aweisberg

File: website/blog/2021-06-04-running-presto-alluxio-sds.md [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+authorURL: https://www.linkedin.com/in/baolong-mao-180624136/
+---
+
+**Tencent**: Baolong Mao
+**Fudan University**: Bowen Ding
+
+## Overview
+
+When running Presto on Hive, Hive Metastore (HMS) is by default used to serve the metadata, including the information about locations of Hive partitions and tables. If you want to migrate the data in Hive to another storage system, the metadata must be updated in HMS. However, in many production deployments, HMS is a global shared service that is highly sensitive, any update to the locations would need to be synchronized in all the compute applications that depend on it. As a result, this process is error-prone and may introduce considerable maintenance overhead for this kind of setup.  In this article, we will introduce an effective solution to avoid such overhead and error using Alluxio Structured Data Service (SDS). 
+
```
Comment (aweisberg) [inline website/blog/2021-06-04-running-presto-alluxio-sds.md:14]:
```suggestion
<!--truncate-->
```

File: website/blog/2021-06-04-running-presto-alluxio-sds.md [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+`./bin/alluxio fs ls /catalog/test/tables/person/hive`
+
+## Summary
+
+In this tutorial, we configured Presto to access the metadata stored in Hive Metastore through Alluxio SDS, without making any changes to the underlying Hive Metastore or any other compute engines. This way Presto can benefit from the flexible data storage solution Alluxio offers.
+
+## Looking Forward
+
+Alluxio SDS acts as a proxy between Presto and Hive Metastore, by understanding the semantics of the data it hosts. This unlocks even more possibilities with Alluxio, e.g. converting CSV to Parquet, coalescing small files.
```
Comment (aweisberg) [inline website/blog/2021-06-04-running-presto-alluxio-sds.md:361]:
```suggestion
Alluxio SDS acts as a proxy between Presto and Hive Metastore, by understanding the semantics of the data it hosts. This unlocks even more possibilities with Alluxio, e.g. converting CSV to Parquet, and coalescing small files.
```

**Review by aweisberg — CHANGES_REQUESTED** [review]
Thank you for working on this. I think the information is well presented and it fills a need for getting people up and running quickly.

That said I have some opinions on using the blog for what looks like documentation. Some of these detailed steps you put here would be better being coherently integrated into Presto's documentation (which has some but maybe not all of this), and then the blog could link to both Presto's documentation for the Presto and Hive setup and Alluxio's documentation for the Alluxio setup.

Yes I am selfishly trying to encourage people to work on Presto's documentation :-)

The blog post could then be a more high level description of what this metastore cache and filesystem do, maybe some performance information, and then links to the relevant documentation.

**Comment by wanglinsong** [comment]
@ dbw9580
is this PR still needed?

**Comment by alileclerc** [comment]
@apc999 @beinan does this need to be updated? if you want to make any updates/changes, we can merge this when ready.

**Comment by dbw9580** [comment]
I think Alluxio SDS is deprecated, so this PR is not needed any more. cc @apc999 & @beinan .

**Comment by apc999** [comment]
close this PR

**Pull request closed without merging**